plugins/crashes/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "countly-crashes",
-  "title": "Crash analytics",
+  "title": "Crash analytics (Vue)",
```

Comment:
Well, why `(Vue)`?

Reply:
This was for debugging purposes, it'll be removed before it hits master 😇